launcher/teeserver/tee_server.go

```diff
@@ -133,15 +135,50 @@ func (a *attestHandler) getITAToken(w http.ResponseWriter, r *http.Request) {
 	a.attest(w, r, a.clients.ITA)
 }
 
+func (a *attestHandler) parseVerifyMethod(headers http.Header) agent.VerifyMethod {
+	if headers == nil {
+		a.logger.Info("No headers specified in request.")
+		return agent.VerifyUnset
+	}
+
+	methods, ok := headers[verifyMethodHeader]
```

Comment:
Why do we have the workload specify the attestation type? This feels to me like an implementation detail that the caller should be unaware of